website/content/docs/upgrading/index.mdx

```diff
@@ -56,9 +56,9 @@ do not allow external network connectivity during testing, in case credentials
 expire. This prevents the test cluster from trying to revoke these resources
 along with the non-test cluster.
 
-## OSS to enterprise installations
+## Upgrading from Community to Enterprise editions
 
-Upgrading to Vault Enterprise installations follow the same steps as OSS upgrades except that the Vault Enterprise binary is to be used and the license file [applied](/vault/api-docs/system/license#install-license), when applicable. The Enterprise binary and license file can be obtained through your HashiCorp sales team.
+Upgrading to Vault Enterprise installations follow the same steps as Community edition upgrades except that the Vault Enterprise binary is to be used and the license file [applied](/vault/api-docs/system/license#install-license), when applicable. The Enterprise binary and license file can be obtained through your HashiCorp sales team.
```

Comment:
This instance is written as "Community edition", but I notice that almost all others are written with titlecase as "Community Edition". Could this be a typo?